src/modules/navigator/rtl_params.c

```diff
@@ -89,11 +90,11 @@ PARAM_DEFINE_FLOAT(RTL_DESCEND_ALT, 30);
 PARAM_DEFINE_FLOAT(RTL_LAND_DELAY, -1.0f);
 
 /**
- * Minimum distance to trigger rising to a safe altitude
+ * Maximum horizontal distance to home position below which vehicle will use
```

Comment:
**Much** better but perhaps ...

> Maximum horizontal distance from home, below which RTL_DESCEND_ALT is used as return mode altitude.

>If the vehicle is less than this horizontal distance from home when return mode is activated it will ascend to RTL_DESCEND_ALT for the return journey (rather than the altitude set by RTL_RETURN_ALT and RTL_CONE_ANG).